src/rdb/rdb.c

```diff
@@ -766,6 +766,8 @@ rdb_chkpt_wait(void *arg, uint64_t wait_id, uint64_t *commit_id)
 	struct rdb_chkpt_record *dcr   = &db->d_chkpt_record;
 	struct umem_store       *store = dcr->dcr_store;
 
+	D_DEBUG(DB_MD, DF_DB ": commit >= " DF_X64 "\n", DP_DB(db), wait_id);
```

Comment:
[Nit; question] Do we expect this message (and those newly-added ones below) to be frequent? We should use something like `DB_TRACE`, rather than `DB_MD`, for frequent messages

Reply:
Possibly, yes.  Basically one set of update ones per commit.  The wait messages are relatively infrequent since checkpointing only happens once every ten seconds and rarely waits for a commit.  I could make them TRACE if that's preferred here.  They are not so frequent that it's overwhelming though.